components/SummaryComponents/RepoOverviewTable/ReposOverviewTable.tsx

```diff
@@ -1,56 +1,77 @@
 import Table from "@mui/material/Table";
-import TableCell from "@mui/material/TableCell";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
 import TableRow from "@mui/material/TableRow";
 import styles from "./ReposOverviewTable.module.css";
 import RedIcon from "../../../components/images/redIcon.svg";
 import YellowIcon from "../../../components/images/yellowIcon.svg";
 import greenIcon from "../../../components/images/greenIcon.svg";
 import Image from "next/image";
+import { TableBody } from "@mui/material";
+
+type RowProps = {
+  icon: any;
+  iconAlt?: string;
+  statusCount: number;
+  statusLabel: string;
+};
+
+enum StatusLabel {
+  RED = "need updating now",
+  YELLOW = "should be updated soon",
+  GREEN = "up-to-date",
+}
+
+const Row = (props: RowProps) => {
+  const ICON_SIZE = "40px";
+  return (
+    <TableRow>
+      <TableCell className={styles.tableCellStyleIcon}>
+        <Image
+          src={props.icon}
+          alt={props.iconAlt ? props.iconAlt : ""}
+          width={ICON_SIZE}
+          height={ICON_SIZE}
+        ></Image>
+      </TableCell>
+      <TableCell className={styles.tableCellStyleCount}>
+        <p>{props.statusCount} </p>
+      </TableCell>
+      <TableCell className={styles.statusLabel}>
```

Comment:
Not sure exactly where to put this comment but I think that the status style labels could have a larger font size as well; on my browser it looks kinda small and I feel like it could take up more room. 
<img width="1183" alt="image" src="https://user-images.githubusercontent.com/81656054/188252962-263aa640-f22f-4bdf-8eff-7868c92a6c8c.png">